### PR TITLE
Add KISS Linux logo

### PR DESCRIPTION
**Pull request by draumaz**
Logo is once again from @dylanaraps' neofetch.

As a sidenote; I noticed while compiling on KISS, ```#include <sys/time.h>``` needs to be included in src/common/networking.c for a successful compile. 

While I've only got evidence of the compile failure happening under KISS (perhaps as a side-effect of being musl based, though this behavior isn't reproducible on something like Alpine), is this patch something that might be useful to implement into master? 

Below is the failed compile log.

```
/home/draumaz/kiss-logo/src/common/networking.c: In function 'ffNetworkingGetHtt':
/home/draumaz/kiss-logo/src/common/networking.c:29:24: error: storage size of 'timev' isn't known
   29 |         struct timeval timev;
      |                        ^~~~~
/home/draumaz/kiss-logo/src/common/networking.c:29:24: warning: unused variable timev' [-Wunused-variable]
make[2]: *** [CMakeFiles/libfastfetch.dir/build.make:272: CMakeFiles/libfastfetch.dir/src/common/networking.c.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:87: CMakeFiles/libfastfetch.dir/all] Error 2
make: *** [Makefile:156: all] Error 2
```

### CODE REVIEW

File: src/logo/builtin.c [hunk 1]
```diff
@@ -775,6 +775,26 @@ static const FFlogo* getLogoKDENeon()
...
+    FF_LOGO_INIT
+    FF_LOGO_NAMES("kiss", "kiss-linux", "kisslinux")
+    FF_LOGO_LINES(
+        "    ___      \n"
+        "   (.· |     \n"
+        "   (<> |     \n"
+        "  / __  \\    \n"
+        " ( /  \\ /|   \n"
+        "_/\\ __)/_)   \n"
+        "\\/-____\\/    \n"
```
Comment (LinusDierheimer) [inline src/logo/builtin.c:789]: The logo is solely magenta, add $2 to change to blue and $1 to change back to magenta. The neofetch logo also has some white in it.

**Review by LinusDierheimer — COMMENTED** [review]
And please add Kiss to the README.

**Comment by LinusDierheimer** [comment]
The time include is fixed.

File: src/logo/builtin.c (hunk 1 above)
Comment (draumaz) [reply src/logo/builtin.c]: Fixed

**Comment by LinusDierheimer** [comment]
Thanks, looking good!

**Pull request merged**